pkg/rotator/rotator.go

```diff
@@ -578,25 +581,26 @@ func (r *ReconcileWH) Reconcile(request reconcile.Request) (reconcile.Result, er
 	return reconcile.Result{}, nil
 }
 
-func (r *ReconcileWH) ensureVWHCerts(certPem []byte) error {
-	vwh := &unstructured.Unstructured{}
-	vwh.SetGroupVersionKind(vwhGVK)
-	if err := r.client.Get(r.ctx, r.vwhKey, vwh); err != nil {
-		if k8sErrors.IsNotFound(err) {
-			crLog.Info("VWK " + r.vwhKey.Name + " is not found. No action is needed.")
-			return nil
+func (r *ReconcileWH) ensureWHCerts(certPem []byte) error {
+	for _, webhook := range r.webhooks {
+		wh := &unstructured.Unstructured{}
+		wh.SetGroupVersionKind(webhook.GVK)
+		if err := r.client.Get(r.ctx, webhook.Name, wh); err != nil {
+			if k8sErrors.IsNotFound(err) {
+				crLog.Info("VWK " + webhook.Name.Name + " is not found. No action is needed.")
```

Comment:
The "V" in "VWK" appears to stand for "validating." Maybe rewrite this and correctly use structured logging?

```
crLog.Info("Webhook not found, no action needed", "gvk", webhook.GVK, "name", webhook.Name.Name")
```

Reply:
Done